styles.css

```diff
@@ -459,6 +467,46 @@ header {
   nav {
     display: none;
   }
+
+  .show-menu {
+    transform: translateX(-100%);
+  }
+
+  .mobile-menu {
+    z-index: 1;
+    position: absolute;
```

Comment:
- [ ] [OPTIONAL] I think it would be great if you use `position: fixed;` to make the mobile menu stick even when the page underneath is scrolling. I believe that will result in a better user experience. 🤓️